package.json

```
@@ -94,6 +95,7 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
+    "@callstack/react-native-visionos": "^0.0.0-0 || 0.73",
```

Does `@callstack/react-native-visionos` publish nightlies?
```suggestion
    "@callstack/react-native-visionos": "0.73",
```
Also, we should make it optional.